app/css/components/youtube-container.css

```diff
@@ -0,0 +1,16 @@
+.c-youtube-container {
+    @apply relative rounded-8 overflow-hidden;
+    padding-bottom: 56.25%;
+    padding-top: 30px;
+    height: 0;
```

Comment:
This is nice, it is used in many places, so I'm glad this class will replace the handcrafted containers.